apps/frontend/src/components/global/checklist/ChecklistRuleEdit.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <v-card class="mt-3 pt-4" height="42vh">
+    <v-card-text>
+      <v-row>
+        <v-col>
+          <v-select
+            v-model="selectedRule.status"
+            dense
+            label="Status"
+            :items="statusItems"
+            item-text="name"
+            item-value="value"
+            :item-color="statusColor(selectedRule.status)"
+          />
+        </v-col>
+        <v-col>
+          <v-select
+            v-model="selectedRule.severityOverride"
+            dense
+            label="Severity Override"
+            item-text="name"
+            item-value="value"
+            :items="checkPossibleOverrides(selectedRule.severity)"
+            @change="promptSeverityJustification"
+          />
+        </v-col>
+      </v-row>
+      <v-row class="mt-n8">
+        <v-col>
+          <strong>Finding Details: </strong><br />
+          <v-textarea
+            v-model="selectedRule.findingDetails"
+            solo
+            outlined
+            dense
+            no-resize
```

Comment:
something to explore: does this actually affect things?

Reply:
It does.  It seems like when in a "full build" and not a "front end only build" some of the frontend changes won't apply while running